sqlparse/engine/filter_stack.py

```diff
@@ -17,10 +17,10 @@ def __init__(self):
         self.preprocess = []
         self.stmtprocess = []
         self.postprocess = []
-        self._grouping = False
+        self.grouping_filter = filters.GroupingFilter()
 
     def enable_grouping(self):
-        self._grouping = True
+        self.grouping_filter.enable()
```

Comment:
I was working off of this change today and found myself wanting to initialize it to None on line 20, then build the GroupingFilter and append it it to stmtprocess here if it is None.